Update zha.markdown with info on visualization (#26848)
home-assistant/home-assistant.github.io: source/_integrations/zha.markdown

  - Place your Zigbee Coordinator away from any Wi-Fi access points and all other sources of WiFi.
  - Wi-Fi frequency ranges can overlap with Zigbee, see the section above on defining Zigbee channel use.

### Zigbee network visualization in ZHA UI

> ZHA configuration UI has a window/tab for "Visualization" that provides a visual representation of device links in your Zigbee network topology.

Reviewer: ```suggestion
ZHA configuration UI has a window/tab for *Visualization* that provides a visual representation of device links in your Zigbee network topology.
```
Use Markdown syntax for highlighting words.

> ZHA configuration UI has a window/tab for *Visualization* that provides a visual representation of device links in your Zigbee network topology.

Reviewer 2: ```suggestion
ZHA configuration UI has a window/tab for *Visualization* that visually represents device links in your Zigbee network topology.
```

> The network visualization can help to get a quick overview in identifying devices that are reporting links with poor connection (i.e. low values). However, be aware that for deeper troubleshooting you will need to look at the ZHA logs instead.

Reviewer 2: ```suggestion
The network visualization can help get a quick overview in identifying devices reporting links with poor connection (that is, low values). However, be aware that you will need to look at the ZHA logs instead for deeper troubleshooting.
```

> This network visualization can sometimes be of help to get a quick overview in identifying devices that are reporting links with poor connection (e.i. low values), however, be aware that for deeper troubleshooting you will need to look at the ZHA logs instead.

Reviewer: ```suggestion
The network visualization can help to get a quick overview in identifying devices that are reporting links with poor connection (i.e. low values). However, be aware that for deeper troubleshooting you will need to look at the ZHA logs instead.
```

> The visualization tried to show multi-hop connections between your paired devices and their reported cumulative values of RSSI (“Received Signal Strength Indicator") and LQI ("Link Quality Indication").

Reviewer: ```suggestion
The visualization shows multi-hop connections between your paired devices and their reported cumulative values of Received Signal Strength Indicator (RSSI) and Link Quality Indication (LQI).
```

> Note that the exact method in which these values are reported depends on the Zigbee network stack being used on each device, and LQI values can be modified at each step as the massage propagates through the mesh networking matrix.

Reviewer 2: you can drop `being` from `network stack being used`

Author: OK, done.

> Note that the exact method in which these values are reported depends on the Zigbee network stack being used on each device, and LQI values can be modified at each step as the massage propagates through the mesh networking matrix.

Reviewer: Replace this with the <div class='note'></div> standard note section.

Author: @dannytsang  standard note section? No other notes in ZHA integration documentation use such, can be seperate PR?

Reviewer: Yeah... fair enough. Don't forget to mark it ready for review when you're ready.

Author: OK thank you for the review. I will mark this conversation as resolved and then mark this PR ready for review again.

  - Place your Zigbee Coordinator away from any Wi-Fi access points and all other sources of WiFi.
  - Wi-Fi frequency ranges can overlap with Zigbee, see the section above on defining Zigbee channel use.

### Zigbee network visualization in ZHA UI

> ZHA configuration UI has a window/tab for *Visualization* that visually represents device links in your Zigbee network topology.

Reviewer 3: ```suggestion
The ZHA configuration UI has a tab to visualize device links in your Zigbee network topology.
```

> The network visualization can help to identify devices with poor connection (that is, low values on the link). To find more detailed information required for troubleshooting, you will need to look at the ZHA logs.

Reviewer 2: ```suggestion
The network visualization can help to identify devices with poor connection (that is, low values on the link). You will need to look at the ZHA logs to find more detailed information required for troubleshooting.
```

> The network visualization can help get a quick overview in identifying devices reporting links with poor connection (that is, low values). However, be aware that you will need to look at the ZHA logs instead for deeper troubleshooting.

Reviewer 3: ```suggestion
The network visualization can help to identify devices with poor connection (that is, low values on the link). To find more detailed information required for troubleshooting, you will need to look at the ZHA logs.
```

The visualization shows multi-hop connections between your paired devices and their reported cumulative values of Received Signal Strength Indicator (RSSI) and Link Quality Indication (LQI).

> The exact method in which these values are reported depends on the Zigbee network stack used on each device. LQI values can be modified at each step as the massage propagates through the mesh networking matrix.

Reviewer 2: ```suggestion
The exact method in which these values are reported depends on the Zigbee network stack used on each device. LQI values can be modified at each step as the message propagates through the mesh networking matrix.
```

> Note that the exact method in which these values are reported depends on the Zigbee network stack used on each device, and LQI values can be modified at each step as the massage propagates through the mesh networking matrix.

Reviewer 3: ``` suggestion
The exact method in which these values are reported depends on the Zigbee network stack used on each device. LQI values can be modified at each step as the massage propagates through the mesh networking matrix.
```

#### How to interpret RSSI and LQI values

> Interpreting RSSI and LQI values is complex. Unless you are Zigbee specialist yourself or are guided by one, please ignore those values. They can be misleading.  If you do delve into this, then it is important to understand not to judge RSSI or LQI values on their own. When troubleshooting Zigbee messages that are being dropped, you must interpret the combination of both RSSI and LQI.

Reviewer 2: ```suggestion
Interpreting RSSI and LQI values is complex. Unless you are a Zigbee specialist yourself or are guided by one, please ignore those values. They can be misleading. If you delve into this, it is important to understand not to judge RSSI or LQI values on their own. When troubleshooting Zigbee messages that are being dropped, you must interpret the combination of both RSSI and LQI.
```

> Interpreting RSSI and LQI values is complicated and the recommendation is that normal users should ignore them unless guided by Zigbee troubleshooting specialists. If you do delve into this then it is important to understand not to judge only RSSI or LQI values on their own separately as it is really the combination of these two that needs to interpret be when troubleshooting if Zigbee messages are being dropped.

Reviewer 3: ```suggestion
Interpreting RSSI and LQI values is complex. Unless you are Zigbee specialist yourself or are guided by one, please ignore those values. They can be misleading.  If you do delve into this, then it is important to understand not to judge RSSI or LQI values on their own. When troubleshooting Zigbee messages that are being dropped, you must interpret the combination of both RSSI and LQI.
```

> RSSI (Received Signal Strength Indicator) values are an indicator value of the raw signal strength between two devices. RSSI values are negative numbers in -dBm format (0 to -100 power ratio in decibels of the measured power referenced to one milliwatt). Lower negative values indicate less interference and a good signal. RSSI information is only between the endpoint device and the first hop from that device. As such it may not necessarily show signal strength to the Zigbee Coordinator but instead could be showing signal strength to the nearest Zigbee Router device.

Reviewer 2: ```suggestion
RSSI (Received Signal Strength Indicator) values are an indicator value of the raw signal strength between two devices. RSSI values are negative numbers in -dBm format (0 to -100 power ratio in decibels of the measured power referenced to one milliwatt). Lower negative values indicate less interference and a good signal. RSSI information is only between the endpoint device and the first hop from that device. As such, it may not necessarily show signal strength to the Zigbee Coordinator but instead could be showing signal strength to the nearest Zigbee Router device.
```

> RSSI (Received Signal Strength Indicator) values are simple to interpret as they are just an indicator value of the raw signal strength between two devices. RSSI values are shown as negative numbers represented in -dBm format (0 to -100 power ratio in decibels of the measured power referenced to one milliwatt). Lower negative values are better as they indicate less interference and a good signal. It is important to understand that RSSI information is only between the endpoint device and the first hop from that device, as such it may not necessarily show signal strength to the Zigbee Coordinator but instead could be showing signal strength to the nearest Zigbee Router device.

Reviewer 3: ```suggestion
RSSI (Received Signal Strength Indicator) values are an indicator value of the raw signal strength between two devices. RSSI values are negative numbers in -dBm format (0 to -100 power ratio in decibels of the measured power referenced to one milliwatt). Lower negative values indicate less interference and a good signal. RSSI information is only between the endpoint device and the first hop from that device. As such it may not necessarily show signal strength to the Zigbee Coordinator but instead could be showing signal strength to the nearest Zigbee Router device.
```

- Generally, anything -60 and above (meaning -50, -40, etc.) in RSSI should be considered a strong signal (not losing messages).

> - Anything at -80 and below ( meaning -85, -90, etc.) should be considered a noise environment and you risk losing messages.

Reviewer 3: ```suggestion
- Anything at -80 and below (meaning -85, -90, etc.) should be considered a noise environment and you risk losing messages.
```

> LQI (Link Quality Index) values can be hard to interpret for Zigbee. This is because the Zigbee specifications and the (IEEE 802.15.4 specification) do not standardize how to perform LQI measurements. LQI values are shown as positive numbers on a scale. But because the value provided by the Zigbee devices is not measured using unified standards from all device manufacturers and Zigbee stacks the values can not always be trusted. For example, Zigbee devices based on Silicon Labs EmberZNet stack use positive display numbers for LQI, where higher is better and lower is worse. The Texas Instruments Z-Stack computes LQI for each received packet from the raw “received signal strength index” (RSSI) by linearly scaling it between the minimum and maximum defined RF power levels for the radio, that more or less just provide an LQI value that based on the strength of the received signal.  This can be misleading in  case the user have a noisy environment with interference within the same frequency range (as the RSSI value may be shown as increased even though the true link quality decreases). And other manufacturers and Zigbee stacks measure and calculate LQI values in another way.

Reviewer 2: ```suggestion
LQI (Link Quality Index) values can be hard to interpret for Zigbee. This is because the Zigbee specifications and the (IEEE 802.15.4 specification) do not standardize how to perform LQI measurements. LQI values are shown as positive numbers on a scale. But because the value provided by the Zigbee devices is not measured using unified standards from all device manufacturers and Zigbee stacks, the values can not always be trusted. For example, Zigbee devices based on Silicon Labs EmberZNet stack use positive display numbers for LQI, where higher is better and lower is worse. The Texas Instruments Z-Stack computes LQI for each received packet from the raw “received signal strength index” (RSSI) by linearly scaling it between the minimum and maximum defined RF power levels for the radio that more or less just provides an LQI value that, based on the strength of the received signal.  This can be misleading in case the user has a noisy environment with interference within the same frequency range (as the RSSI value may be shown as increased even though the true link quality decreases). Other manufacturers and Zigbee stacks measure and calculate LQI values in another way.
```

> - In theory, an LQI value 255 means a zero error rate in theory. And in general, a positive high LQI value is better and a lower LQI value is worse. However, depending on your devices that might not be the reality.

Reviewer 2: ```suggestion
- In theory, an LQI value of 255 means a zero error rate in theory. In general, a positive high LQI value is better and a lower LQI value is worse. However, depending on your devices, that might not be the reality.
```

> LQI (Link Quality Index) values can be hard to interpret for Zigbee. The reason for this is that how to perform LQI measurements are not standardized as part of the Zigbee specifications (and the IEEE 802.15.4 specification). LQI values are shown as positive numbers on a scale, but because the information value provided by the Zigbee devices is not measured using unified standards from all device manufacturers and Zigbee stacks it can not always be trusted as truth. For example; Zigbee devices based on Silicon Labs EmberZNet stack use positive display numbers for LQI, where higher is better and lower is worse, while the Texas Instruments Z-Stack, on the other hand, instead computes LQI for each received packet from the raw “received signal strength index” (RSSI) by linearly scaling it between the minimum and maximum defined RF power levels for the radio, that more or less just provide an LQI value that based on the strength of the received signal, and that can be misleading in the case the user have a noisy environment with interference within the same frequency range (as the RSSI value may be shown as increased even though the true link quality decreases). And other manufacturers and Zigbee stacks measure and calculate LQI values in yet different ways.

Reviewer 3: ```suggestion
LQI (Link Quality Index) values can be hard to interpret for Zigbee. This is because the Zigbee specifications and the (IEEE 802.15.4 specification) do not standardize how to perform LQI measurements. LQI values are shown as positive numbers on a scale. But because the value provided by the Zigbee devices is not measured using unified standards from all device manufacturers and Zigbee stacks the values can not always be trusted. For example, Zigbee devices based on Silicon Labs EmberZNet stack use positive display numbers for LQI, where higher is better and lower is worse. The Texas Instruments Z-Stack computes LQI for each received packet from the raw “received signal strength index” (RSSI) by linearly scaling it between the minimum and maximum defined RF power levels for the radio, that more or less just provide an LQI value that based on the strength of the received signal.  This can be misleading in  case the user have a noisy environment with interference within the same frequency range (as the RSSI value may be shown as increased even though the true link quality decreases). And other manufacturers and Zigbee stacks measure and calculate LQI values in another way.
```

> - LQI value 255 means a zero error rate in theory, a positive high LQI value is better and a lower LQI value is worse, however, depending on your devices that might not be the reality.

Reviewer 3: ```suggestion
- In theory, an LQI value 255 means a zero error rate in theory. And in general, a positive high LQI value is better and a lower LQI value is worse. However, depending on your devices that might not be the reality.
```
